pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -203,6 +207,7 @@ func (gc *GarbageCollector) Sync(discoveryClient discovery.ServerResourcesInterf
 				newResources = GetDeletableResources(discoveryClient)
 				if len(newResources) == 0 {
 					klog.V(2).Infof("no resources reported by discovery (attempt %d)", attempt)
+					metrics.GarbageCollectorResourcesSyncError.WithLabelValues("resource_discovery_failure").Inc()
```

Comment:
Can you define the label values as const in the metrics package?

Reply:
done